.travis.yml

```diff
@@ -9,19 +9,16 @@ sudo: false
 
 python:
   - 3.5
+  - 3.8
 
 env:
-  - TOX_ENV=django111
-  - TOX_ENV=django20
-  - TOX_ENV=django21
   - TOX_ENV=django22
+  - TOX_ENV=js
+  - TOX_ENV=quality
 
-matrix:
-  include:
-    - python: 3.5
-      env: TOX_ENV=js
-    - python: 3.5
-      env: TOX_ENV=quality
+jobs:
```

Comment:
Just for curiosity, why it was changed to jobs?

Reply:
Removed the matrix part and just using allow_failures only.